### PR TITLE
DOCSP-44902 TOC Relabel

### PR DESCRIPTION
**Pull request by lindseymoore**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kotlin/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-44902>
Staging - <https://deploy-preview-179--docs-kotlin.netlify.app/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/builders.txt [hunk 1]
```diff
@@ -6,12 +6,12 @@ Builders
...
-   /fundamentals/builders/indexes
-   /fundamentals/builders/projections
-   /fundamentals/builders/sort
-   /fundamentals/builders/updates
+   Aggregates </fundamentals/builders/aggregates>
+   Filters </fundamentals/builders/filters>
+   Indexes </fundamentals/builders/indexes>
+   Projections </fundamentals/builders/projections>
+   Sorts </fundamentals/builders/sort>
+   Updates </fundamentals/builders/updates>
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:14]: Same suggestion as i made for Java

File: source/fundamentals/crud.txt [hunk 2]
```diff
@@ -7,10 +7,10 @@ CRUD Operations
 .. toctree::
    :caption: CRUD Operations
 
-   /fundamentals/crud/read-operations
-   /fundamentals/crud/write-operations
-   /fundamentals/crud/query-document
-   /fundamentals/crud/compound-operations
+   Read Operations </fundamentals/crud/read-operations>
+   Write Operations </fundamentals/crud/write-operations>
```
Comment (mongoKart) [inline source/fundamentals/crud.txt:11]: S: Remove Operations

File: source/usage-examples.txt [hunk 3]
```diff
@@ -10,16 +10,15 @@ Usage Examples
...
-   /usage-examples/bulkWrite
-   /usage-examples/watch
-   /usage-examples/count
-   /usage-examples/distinct
-   /usage-examples/command
-
+   Find Operations </usage-examples/find-operations>
+   Insert Operations </usage-examples/insert-operations>
+   Update & Replaces Operations </usage-examples/update-operations>
+   Delete Operations </usage-examples/delete-operations>
```
Comment (mongoKart) [inline source/usage-examples.txt:16]: S: Remove "Operations"

**Review by mongoKart — APPROVED** [review]
lgtm w/ suggestions

**Review by sarahemlin — CHANGES_REQUESTED** [review]
LGTM with Mike's changes

**Review by sarahemlin — APPROVED** [review]

**Pull request merged**